src/modules/auto-toggle-class-on-scroll.js

```diff
@@ -20,9 +20,9 @@
 	var ATTR_PREFIX = 'data-toggle-class-on-scroll-';
 
 	var ATTR_ADD_BEFORE = ATTR_PREFIX + 'add-before';
-	var ATTR_REM_BEFORE = ATTR_PREFIX + 'rem-before';
+	var ATTR_REM_BEFORE = ATTR_PREFIX + 'remove-before';
```

Comment:
Good point @fhamon . @PascalPiche we need to choose one or the other and always use the same. I vote for 'rem'